prometheus-ksonnet/jsonnetfile.json

```diff
@@ -8,7 +8,7 @@
                     "subdir": ""
                 }
             },
-            "version": "release-0.1"
+            "version": "master"
```

Comment:
kubernetes-mixin@master only works with k8s 1.14 or newer.   the GKE stable release is still 1.13.x.

Reply:
so instead of using master, i just submitted a PR to fix the "release-0.1" branch instead.

https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/332

Reply:
Thank you! Somehow assumed `release-0.1` was a tag, not a branch

Reply:
Why does it only work with 1.14 or newer?

Reply:
> Why does it only work with 1.14 or newer?

https://github.com/kubernetes-monitoring/kubernetes-mixin#releases

Reply:
Okay. So I assume we can close this?